packages/features/bookings/Booker/components/Header.tsx

```diff
@@ -0,0 +1,111 @@
+import { ChevronLeft, ChevronRight } from "lucide-react";
```

Comment:
Whole file looks new, but main part is actually a rename of the `LargeViewHeader` component below. Only added the layout toggle and did a few tweaks.